### PR TITLE
feat: Slash Command Parser

### PR DESCRIPTION
**Pull request by sdelamo**

### CODE REVIEW

File: chatbots-telegram-core/src/main/java/io/micronaut/chatbots/telegram/core/TelegramSlashCommandPaser.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        }
+        return Optional.empty();
+    }
+
+    @NonNull
+    private Optional<String> parse(@NonNull Message message) {
+        String text = message.getText();
+        if (StringUtils.isNotEmpty(text)) {
+            int index = text.indexOf("/");
+            if (index != -1) {
```
Comment (timyates) [inline chatbots-telegram-core/src/main/java/io/micronaut/chatbots/telegram/core/TelegramSlashCommandPaser.java:51]: Shouldn't this be `if (index == 0)`? as the slash must be the first character?

File: chatbots-telegram-core/src/main/java/io/micronaut/chatbots/telegram/core/TelegramSlashCommandPaser.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                String result = text.substring(index);
+                index = result.indexOf("@");
+                if (index != -1) {
+                    result = text.substring(0, index);
+                }
```
Comment (timyates) [inline chatbots-telegram-core/src/main/java/io/micronaut/chatbots/telegram/core/TelegramSlashCommandPaser.java:56]: What's this for?  I've not seen a `/something whatever @Karen` slash-command

Is that a thing?

Comment (timyates) [reply chatbots-telegram-core/src/main/java/io/micronaut/chatbots/telegram/core/TelegramSlashCommandPaser.java:56]: Ahh I see now...

And do we want `/test      @now` to return a result of `test      `, or do we want to trim the result?

File: chatbots-telegram-core/src/main/java/io/micronaut/chatbots/telegram/core/TelegramSlashCommandPaser.java (hunk 1 above)
Comment (sdelamo) [reply chatbots-telegram-core/src/main/java/io/micronaut/chatbots/telegram/core/TelegramSlashCommandPaser.java:51]: what I am checking if there is a slash. 

Not sure if slash will be always the first character or you can send a message such as: 

`@Bot /help`

File: chatbots-telegram-core/src/main/java/io/micronaut/chatbots/telegram/core/TelegramSlashCommandPaser.java (hunk 2 above)
Comment (sdelamo) [reply chatbots-telegram-core/src/main/java/io/micronaut/chatbots/telegram/core/TelegramSlashCommandPaser.java:56]: I think we should not trim it. You can send a command with arguments.

**Review by timyates — APPROVED** [review]

**Pull request merged**